linux/ibus-keyman/src/engine.c

```diff
@@ -277,12 +277,16 @@ reset_context(IBusEngine *engine) {
       if (km_kbp_context_items_from_utf8(surrounding_text, &context_items) == KM_KBP_STATUS_OK) {
         km_kbp_context_set(context, context_items);
         km_kbp_context_items_dispose(context_items);
+      } else {
+        km_kbp_context_clear(context);
+        g_message("%s: setting context failed", __FUNCTION__);
```

Comment:
It would be good to also output the status that `km_kbp_context_items_from_utf8` returned.

Reply:
I've now added the status code to the output.